### PR TITLE
Multiple changes to CONTRIBUTING.md.

### PR DESCRIPTION
**Pull request by flexibeast**
* Describe using `xbps-src pkg` to check that a package builds.
* Move mention of `xlint` to the 'Creating ...' section.
* Fix grammar and typos.
* Improve readability.

### CODE REVIEW

**Comment by flexibeast** [comment]
i've retained the part where it says:

> GitHub reports new pull requests on our IRC channel

but is that still correct? Assuming the channel in question is #voidlinux, i've not seen such messages, but perhaps there's a separate IRC channel for maintainers?

**Comment by Johnnynator** [comment]
> but is that still correct? Assuming the channel in question is #voidlinux, i've not seen such messages, but perhaps there's a separate IRC channel for maintainers?

This refers to #xbps. But it is not correct anymore and can be removed. GitHub did disable their IRC bot some time ago.

**Comment by flexibeast** [comment]
@Johnnynator: Thanks - PR updated.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,62 +1,73 @@
...
-[manual about xbps-src](https://github.com/void-linux/void-packages/blob/master/README.md) which is used
+on how to create new packages. There's also a
+[manual for xbps-src](https://github.com/void-linux/void-packages/blob/master/README.md), which is used
 to build package files from templates.
 
-For this guide, we assume you have basic knowledge about [git](http://git-scm.org) and a [GitHub Account](http://github.com)
+For this guide, we assume you have basic knowledge about [git](http://git-scm.org), as well as a [GitHub Account](http://github.com).
 
-Please note that we do not accept any packages containing non-release versions such
-as specific git- or svn-revisions anymore.
```
Comment (Chocimier) [inline CONTRIBUTING.md]: This "anymore" is significant, there are still previously added snapshots in repo.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,62 +1,73 @@
...
 
+After committing your changes, please check that the package builds successfully. From the top level directory of your local copy of the `void-packages` repository, run:
+
+    $ ./xbps-src pkg <pkgname>
+    
+Your package must build successfully for at least x86, but we recommend trying to build for armv* as well, e.g.:
+
+    $ ./xbps-src -a armv7l pkg <pkgname>
+    
+For further details, see the output of `./xbps-src --help`.
```
Comment (Chocimier) [inline CONTRIBUTING.md]: unrecognized option

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,38 +77,31 @@ When you make changes to your pull request, please *do not close and reopen your
...
-Once you have applied all comments, the reviewers will merge your request.
+Once you have applied all changes requested in reviewer comments, the reviewers will merge your request.
 
-If the pull request gets inactive for some days, the reviewers may or may not warn you when they are about to close it.
-If it stays inactive further, it'll be closed.
+If the pull request becomes inactive for some days, the reviewers may or may not warn you when they are about to close it.
+If it stays inactive further, it will be closed.
 
-Please abstain from temporary closing a pull request while revising the templates. Only close your pull request if
+Please abstain from temporarily closing a pull request while revising the templates. Only close your pull request if
```
Comment (Chocimier) [inline CONTRIBUTING.md]: While touching this, please add what to do instead of closing - commenting what's wrong or adding "[WIP]" to PR title.

**Comment by xtraeme** [comment]
I agree.

File: CONTRIBUTING.md (hunk 1 above)
Comment (flexibeast) [reply CONTRIBUTING.md]: Ah okay, thanks, i'll re-add that.

File: CONTRIBUTING.md (hunk 2 above)
Comment (flexibeast) [reply CONTRIBUTING.md]: Ah, i was fooled by `xbps-src` (very reasonably) showing help output when presented with an unrecognised option. :-) i'll change it to `-h`.

File: CONTRIBUTING.md (hunk 3 above)
Comment (flexibeast) [reply CONTRIBUTING.md]: Thanks - i'll change that to:

> the templates. Instead, leave a comment on the PR describing what still needs work, or add "[WIP]" to the PR title. Only ...

**Comment by flexibeast** [comment]
@Chocimier, @xtraeme:

Well, i'm a native English speaker myself, and apart from "will comment your on pull request", which was a typo i've now fixed ("will comment on your pull request"), these all sound fine to me. That said:

* i can change "the necessary definitions" back to "the definitions" (although the former still sounds better to me);

* "file(s)" is a common way of compactly combining the singular and plural, but i can change that to "file or files";

* i can change "Once you have applied all changes requested in reviewer comments, the reviewers will merge your request" to "Once you have applied all requested changes, the reviewers will merge your request".

Does all that sound okay?

**Comment by Chocimier** [comment]
okay

**Comment by flexibeast** [comment]
@Chocimier: Thanks - PR updated.

**Comment by flexibeast** [comment]
Ping?

**Comment by flexibeast** [comment]
Ping?

**Comment by flexibeast** [comment]
Ping? Is there anything else i need to do for this PR to be merged?

**Comment by xtraeme** [comment]
Looks good to me. Can you please rebase?

**Comment by flexibeast** [comment]
@xtraeme: Thanks, rebased.

**Pull request merged**